pom.xml

```diff
@@ -40,7 +40,7 @@
         <version.org.fenixedu.messaging>4.0.1</version.org.fenixedu.messaging>
         <version.org.glassfish.jersey>2.27</version.org.glassfish.jersey>
         <version.pt.ist.drive-sdk>2.1.0</version.pt.ist.drive-sdk>
-        <version.pt.ist.fenixedu-id-cards>5.0.0</version.pt.ist.fenixedu-id-cards>
+        <version.pt.ist.fenixedu-id-cards>5.3.2</version.pt.ist.fenixedu-id-cards>
```

Comment:
This version should have https://github.com/ist-dsi/fenixedu-id-cards/pull/41

Reply:
Will update to 5.4.0